build.xml

```diff
@@ -295,6 +313,44 @@
 
     </target>
 
+    <target name="lang.docinfo.texi">
+
+        <mkdir dir="${build.dir}/${lang}/info/"/>
+
+        <java classname="com.icl.saxon.StyleSheet" fork="true" dir="${basedir}" maxmemory="192m">
+            <classpath refid="lib.classpath"/>
+            <arg value="-o"/>
+            <arg value="${build.dir}/${lang}/info/ceylon-spec.texi"/>
+            <arg value="${basedir}/${lang}/master.xml"/>
+            <arg value="${basedir}/${lang}/styles/${db.style.info}"/>
+        </java>
+
+    </target>
+
+    <target name="lang.docinfo.make" if="hasInfo">
+
+        <exec executable="makeinfo" dir="${build.dir}/${lang}/info/">
+            <arg value="ceylon-spec.texi"/>
+        </exec>
+      
+    </target>
+
+    <target name="lang.docinfo"
+            depends="lang.docinfo.texi,lang.docinfo.make"
+            description="Generates the Texinfo documentation only for a language (set lang)"/>
+
+    <target name="lang.docinfo.install"
+            depends="lang.docinfo"
+            if="hasInfo">
+
+        <exec executable="sudo" dir="${build.dir}/${lang}/info/">
```

Comment:
@jvasileff yes, it’s actually called `INFOPATH` :) however, if it’s unset, `info` falls back to a compile-time default (usually `/usr/share/info`), which I don’t think we can determine.

@FroMage Is there a user info path? I can’t find it. Of course, you can make it yourself with `INFOPATH`, but it doesn’t seem to be intended in `info`’s design.